readme.md

```
@@ -82,9 +84,63 @@ part of the deployable release artifact.
+Follow the instructions of your Ide to build and start the container. In VS-Code select "Dev Containers: Rebuild Container".
+
+As result your container gets build and you have access to thses URLs:
+
+- http://localhost:8000 The Fauxton UI
+- http://localhost:5984 The CouchDB
+
+### Running the devcontainer with keycloak
+
+Since the keycloak IdP is not nescesary unless you want to use JWT related operations, it doesn't automatically start. You have to use a terminal/commandline after you started the dev container.
+
+
```

markdownlint-cli2 (0.17.2)

Multiple consecutive blank lines
Expected: 1; Actual: 2

(MD012, no-multiple-blanks)